README.md

```diff
@@ -30,6 +30,9 @@
 ### Usage
 This is the basic usage. If you want to see more you can read [API](/API.md)
 
+> you can input this command `PastImg` in markdown to copy image.
+> By default, it will create a new img folder in the current directory
+
```

Comment:
This was helpful for me. For some reason I expected `p` to "just work" for pasting images.

Perhaps something like this:

---

To paste an image, execute the `:PasteImg` command.

 Alternatively, you can bind it to a key combo. For example,

```vim
nnoremap <leader>p :PasteImg<cr>
```